test/acceptance/element.robot

```diff
@@ -247,4 +246,23 @@ VerifyAttributeNOK
     # not valid css
     Run Keyword And Expect Error      QWebElementNotFoundError:*    VerifyAttribute      //button             value                Button2              element_type=css     timeout=2
     # multiple found css
-    Run Keyword And Expect Error      QWebValueError:*              VerifyAttribute      button               value                Button2              element_type=css     timeout=2
\ No newline at end of file
+    Run Keyword And Expect Error      QWebValueError:*              VerifyAttribute      button               value                Button2              element_type=css     timeout=2
+
+VerifyElementTextBorders
+    [Tags]                      VerifyElementText    DrawBorders
+    Go To                       file://${CURDIR}/../resources/text.html
+    
+    ${no_border_file}           SetVariable            ${CURDIR}/no_border.png
```

Comment:
I can't see these images, should they be included?

Also maybe could just use VerifyIcon here?

Reply:
Those are just filepath strings for screenshots which will be taken during the the Test and there should be no need for stored images

Reply:
To clarify, test takes screenshot from initial state with no element highlights.
Then an element is highlighted and a new screenshot is taken.
The screenshots are then compared on binary level and they should be different.

VerifyIcon would also work but requires an existing image.

I don't actually know which would be better approach 😅

Reply:
ah, okay. That explains why I couldn't find those images :).